substrate/frame/support/src/storage/types/counted_nmap.rs

```diff
@@ -45,6 +45,36 @@ use sp_std::prelude::*;
 ///
 /// Whenever the counter needs to be updated, an additional read and write occurs to update that
 /// counter.
+///
+/// For general information regarding the `#[pallet::storage]` attribute, refer to 
+/// [`crate::pallet_macros::storage`].
+///
+/// # Example
+///
+/// ```
+/// #[frame_support::pallet]
+/// mod pallet {
+///     # use frame_support::pallet_prelude::*;
+///     # #[pallet::config]
+///     # pub trait Config: frame_system::Config {}
+///     # #[pallet::pallet]
+///     # pub struct Pallet<T>(_);
+/// 	/// A kitchen-sink storage map, with all possible additional attributes.
```

Comment:
```suggestion
/// 	/// A kitchen-sink storage N map, with all possible additional attributes.
/// 	/// It uses 3 storage keys of `u8`, `u16` and `u32`
```

Or something like that.